Week2/homework/const.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const numbers = [1, 2, 3, 4];
+
+// Add your code here
+const oddNumbers = numbers.filter(number => {
+    return number % 2 !== 0;
+});
+console.log(oddNumbers);
+
+const newNumbers = oddNumbers.map(number => {
+    return number * 2;
+});
+
+console.log('The doubled numbers are', newNumbers);
+
```

Comment:
Nice job making this testable by breaking everything up into its own constant.
You can also chain the two operations together.
```
numbers.filter(number => number % 2 !== 0)
    .map(number => number * 2);
```